src/platform/site-wide/cta-widget/tests/index.unit.spec.js

```diff
@@ -286,6 +286,7 @@ describe('<CallToActionWidget>', () => {
         <CallToActionWidget
           fetchMHVAccount={d => d}
           isLoggedIn
+          useSSOe
```

Comment:
Is this prop being passed into any real instances of `CallToActionWidget`? Not sure if I'm missing smoething.

Reply:
missed that in cleaning up a refactor. will remove